lib/test_prof/recipes/rspec/let_it_be.rb

```diff
@@ -22,7 +22,11 @@ def modules
     # Use uniq prefix for instance variables to avoid collisions
     # We want to use the power of Ruby's unicode support)
     # And we love cats!)
-    PREFIX = "@😸".freeze
+    if RUBY_ENGINE == 'jruby'
+      PREFIX = "@😸 ".freeze
```

Comment:
LOL. What is the difference?

Reply:
I'm trying to figure out; it works only with extra spaces in JRuby irb, but haven't tried running from a file

Reply:
Nope, nothing works(